relay-server/src/endpoints/outcomes.rs

```diff
@@ -0,0 +1,38 @@
+use actix_web::HttpResponse;
+use serde::{Deserialize, Serialize};
+
+use crate::actors::outcome::OutcomePayload;
+use crate::extractors::{CurrentServiceState, SignedJson};
+use crate::service::ServiceApp;
+
+/// Defines the structure of the HTTP outcomes requests
+#[derive(Deserialize, Debug)]
+struct SendOutcomes {
+    pub outcomes: Vec<OutcomePayload>,
```

Comment:
_nit_: For future-proofing, let's mark this with `#[serde(default)]`. If future downstream Relays no longer send this and send something else instead, at least this will not break.

Reply:
good point